src/Service.Tests/Unittests/ConfigValidationUnitTests.cs

```diff
@@ -2175,15 +2175,15 @@ public void ValidateUniqueRestPathsForEntitiesInConfig(
             DisplayName = "Runtime base-route specified as null for AppService authentication provider passes config validation.")]
         [DataRow(null, "StaticWebApps", false,
             DisplayName = "Runtime base-route specified as null for Static Web Apps authentication provider passes config validation.")]
-        [DataRow("/    ", "StaticWebApps", false,
+        [DataRow("/    ", "StaticWebApps", true, "Runtime base-route updated Path contains white spaces.",
```

Comment:
why this change?

Reply:
This is because after adding the white spaces error check in the config, we would receive error in this test case